### ISSUE TITLE
MP3 bitrate sometimes displayed incorrectly

### ISSUE THREAD

**Issue by aaronk6** [einsteinx2/iSub#59]
The bitrate shown in the player view is sometimes incorrect. It seems to be related with seeking: I can reproduce this easily when seeking in a song that’s currently being streamed.

<img src="https://cloud.githubusercontent.com/assets/446063/22403712/11f465a0-e620-11e6-9e2a-bc0657d7ad6f.PNG" width="160">

_Bitrate shown as 32 kbps (actually 320)_

<img src="https://cloud.githubusercontent.com/assets/446063/22403713/11f84d64-e620-11e6-9178-0a9036241948.PNG" width="160">

_Bitrate shown as 439 kbps (actually 320)_

**Subsonic transcoding command:**

```
/var/subsonic/transcode/ffmpeg -i /mnt/musik/Artists/A/AC_DC/1995 T.N.T. (album)/106 AC_DC - Rocker.mp3 -map 0:0 -b:a 320k -v 0 -f mp3 -
```

Let me know if you’d like me to test this with formats other than MP3 or other MP3 bitrates.

**Comment by einsteinx2** [comment]
Yeah the bitrate display is "live" from the player, so likely I'm not resetting some number in the calculation after seeking. Also I need to make it a rolling average so it doesn't fluctuate so much.

**Comment by einsteinx2** [comment]
This has been improved and I think fixed, but leaving this open to double check.

**Comment by aaronk6** [comment]
Great, I’ll double-check with the next beta build.
